src/routes/Account/containers/AccountContainer.scss

```diff
@@ -1,25 +1,43 @@
-.container {
-  display: flex;
-  flex-direction: row;
+.container, .form, .pane, .settings {
   align-items: center;
+  display: flex;
```

Comment:
Lets use `@extend .flex` so we get all of the stuff added by that default class.